extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergTools.java

```diff
@@ -16,4 +22,72 @@ public static IcebergCatalogAdapter createAdapter(
             final FileIO fileIO) {
         return new IcebergCatalogAdapter(catalog, fileIO);
     }
+
+    /**
+     * <p>
+     * Create an Iceberg catalog adapter for an Iceberg catalog created from configuration properties. These properties
+     * map to the Iceberg catalog Java API properties and are used to create the catalog and file IO implementations.
+     * </p>
+     * <p>
+     * This is a wrapper around {@link CatalogUtil#buildIcebergCatalog(String, Map, Object)} and accepts the same
```

Comment:
By including a link to this specific method, we are somewhat implicitly elevating iceberg-core from an implementation dependency to an api dependency; I'm not opposed to it, but maybe we should be explicit about it from a dependency standpoint. Or, remove the explicit mention of this method.

Reply:
I can remove this, we just need to document our input fully.